src/source/Signaling/LwsApiCalls.c

```diff
@@ -795,69 +909,42 @@ STATUS describeChannelLws(PSignalingClient pSignalingClient, UINT64 time)
     CHK((SERVICE_CALL_RESULT) ATOMIC_LOAD(&pSignalingClient->result) == SERVICE_CALL_RESULT_OK && resultLen != 0 && pResponseStr != NULL,
         STATUS_SIGNALING_LWS_CALL_FAILED);
 
+    retSignal = Signaling_ParseDescribeSignalingChannelResponse(pResponseStr, resultLen, &channelInfo);
+    CHK(retSignal == SIGNALING_RESULT_OK, retSignal);
```

Comment:
These statuses are not part of the convention? I can't seem to find the definition of the SIGNALING_RESULT_OK - nor the definition of the type but the convention is to use STATUS type.

Reply:
Oh, perhaps these are the call results. Hard to guess by the name

Reply:
retSignal is the return value of Signaling_ParseDescribeSignalingChannelResponse, which is provided by external Signaling library. Refer to [Signaling PR](https://github.com/awslabs/amazon-kinesis-video-streams-signaling/pull/3/files#diff-3c0135dfeb4d017dcfb798016d08b6ce3b0b2a5c71b7141c2415224f0ba3173aR61).